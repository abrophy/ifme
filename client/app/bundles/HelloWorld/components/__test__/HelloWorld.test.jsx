import React from 'react';
import HelloWorld from '../HelloWorld';
import renderer from 'react-test-renderer';

test('test helloworld renders', () => {

    const component = renderer.create(
        <HelloWorld name="foo" />
    )

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();

});
