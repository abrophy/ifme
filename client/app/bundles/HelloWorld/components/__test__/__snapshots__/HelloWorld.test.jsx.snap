// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test helloworld renders 1`] = `
<div>
  <h3>
    Hello, 
    foo
    !
  </h3>
  <hr />
  <form>
    <label
      htmlFor="name"
    >
      Say hello to:
    </label>
    <input
      id="name"
      onChange={[Function]}
      type="text"
      value="foo"
    />
  </form>
</div>
`;
